### ISSUE TITLE
Range is sometimes not updating in the database

### ISSUE THREAD

**Issue by Tackoozle** [BentoBoxWorld/BSkyBlock#297]
#### Describe the bug

When you change the range with /bsb range set <Name> <Range> its not updating on the database. Sometimes it updates and sometimes not. 
When i enter the command and do /bsb info, it shows always the right range (cached?).

#### Steps to reproduce the behavior

Use command, reload database or restart server, sometimes it updates, sometimes not
##### BentoBox Version (Mandatory)

```
[20:32:50 INFO]: Running PAPER 1.15.2.
[20:32:50 INFO]: BentoBox version: 1.12.0
[20:32:50 INFO]: Database: MARIADB
[20:32:50 INFO]: Loaded Game Worlds:
[20:32:50 INFO]: bskyblock_world (BSkyBlock): Overworld, [nether_color]Nether, [end_color]End
[20:32:50 INFO]: Loaded Addons:
[20:32:50 INFO]: Biomes 1.6.0.1 (ENABLED)
[20:32:50 INFO]: BSkyBlock 1.12.0 (ENABLED)
[20:32:50 INFO]: Challenges 0.9.0-SNAPSHOT-b400 (ENABLED)
[20:32:50 INFO]: ControlPanel 1.7.0 (ENABLED)
[20:32:50 INFO]: Level 1.9.4-SNAPSHOT-b345 (ENABLED)
[20:32:50 INFO]: Limits 1.9.4-SNAPSHOT-LOCAL (ENABLED)
[20:32:50 INFO]: MagicCobblestoneGenerator 1.9.0-SNAPSHOT-b100 (ENABLED)
[20:32:50 INFO]: Warps 1.9.3-SNAPSHOT-b222 (ENABLED)
```

#### Plugins (Optional)

```
_______
```
 
### Additional context (Optional)

BentoBox Config:
```
    type: MARIADB
    host: localhost
    # Port 3306 is MySQL's default. Port 27017 is MongoDB's default.
    port: 3306
    name: mc_skyblock
    username: username
    password: password
    # How often the data will be saved to file in mins. Default is 5 minutes.
    # This helps prevent issues if the server crashes.
    # Data is also saved at important points in the game.
    backup-period: 10
    # Enable SSL connection to MongoDB, MariaDB, MySQL and PostgreSQL databases.
    # Added since 1.12.0.
    use-ssl: false
```

**Comment by tastybento** [comment]
If won't show in the database until the islands are saved. That occurs when the server is shutdown or periodically based on your database save setting in BentoBox's config.yml.

**Comment by Tackoozle** [comment]
When I change the range of an island and restart the server, the range is not saved. I just put the backup-period on 2 minutes and for now it looks like its working. Do you know if its gonna' be laggy when there are e.g. 400+ islands?

**Comment by tastybento** [comment]
You do not need to change the backup period. The range is saved in RAM and when you restart the server it will be saved to the database. Do you need it to be saved earlier?

**Comment by Tackoozle** [comment]
The range was set to 60:
```
[21:01:12 INFO]: Tacker_ issued server command: /bsb info
[21:01:45 INFO]: Tacker_ issued server command: /bsb range set Tacker_ 120
> stop
[21:01:50 INFO]: Stopping the server
[21:01:50 INFO]: Stopping server
```
After starting the server:
```
> bsb info Tacker_
[21:02:20 INFO]: ========== Island Info ============
[21:02:20 INFO]: UUID: BSkyBlocke7e3a099-0452-49e2-8fa7-cc0e3aedc08c
[21:02:20 INFO]: Owner: Tacker_ (ba9933af-7862-421f-8c68-b0ffb0cbca04)
[21:02:20 INFO]: Last login: Mon Apr 13 21:01:51 CEST 2020
[21:02:20 INFO]: Deaths: 0
[21:02:20 INFO]: Resets: 2 (Max: 3)
[21:02:20 INFO]: Team members:
[21:02:20 INFO]: Tacker_ Owner
[21:02:20 INFO]: Island location: 0,120,500
[21:02:20 INFO]: Island coordinates: -250,0,250 to 249,256,749
[21:02:20 INFO]: Protection range: 60
[21:02:20 INFO]: Largest historical protection range: 150
[21:02:20 INFO]: Protection coordinates: -60,0,440 to 59,256,559
```
Still 60. Maybe I was a little fast, but it also happens, when i wait 15-20 seconds. If I wait longer the backup will save it.

**Comment by tastybento** [comment]
It should save immediately. I just tested and it works fine but I didn't try specifically with MariaDB. How many islands do you have in your database? 
Also, did the player join the game after you started the server but before you did bsb info?

**Comment by Tackoozle** [comment]
> It should save immediately. I just tested and it works fine but I didn't try specifically with MariaDB. How many islands do you have in your database?
> Also, did the player join the game after you started the server but before you did bsb info?

There are 3 island atm (testing before opening for every user on our server). Im the player (Tacker_). 
I just testet it again 3 times: Change range, restart the server, joined the server and do /bsb info.
2 out of 3 times it was not updated after server restart.

**Comment by tastybento** [comment]
If you are joining the server then you might have a permission that is resetting the range when you join. Look at the console log when you join because there will be a note about range when you join. 
Alternatively, don’t join the server and use the console to check the range.

**Comment by Tackoozle** [comment]
So I tried to do everything in the console and its not working. Neither when im online or offline.
Also tried it like you said: "Alternatively, don’t join the server and use the console to check the range."
I changed it ingame and stayed offline after server restart.
When im waiting the backup-period it is saving, but thats not how it should work I think 

I just got one permission: bskyblock.*

**Comment by tastybento** [comment]
Yes, I have confirmed this occurs on MySQL. I now need to find out why. It does not happen on JSON.
